api/client/service/opts.go

```diff
@@ -176,61 +263,28 @@ func (m *MountOpt) Set(value string) error {
 		}
 	}
 
-	for _, field := range fields {
-		parts := strings.SplitN(field, "=", 2)
-		if len(parts) == 1 && strings.ToLower(parts[0]) == "writable" {
-			mount.Writable = true
-			continue
+	var err error
+	switch strings.ToLower(key) {
+	case "name":
```

Comment:
These shouldn't be differentiated by keys. That was not the intent of suggestion and complicates the grammar, immensely.